bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly_de.properties

```diff
@@ -1,53 +1,507 @@
+# binding
```

Comment:
@openhab/add-ons-maintainers There are quite a lot changes in the translation in this PR. If I understood correctly, translations should only be handled in Crowdin, otherwise they will be overwritten in the repo. How do we handle this?

Reply:
Most of the translations are even no changed. I moved all str resources from xml to the .properties file. Changes must not be overwritten depending on tooling. Please advise how to proceed

Reply:
First, the Shelly binding is one of the bindings for which i18n is not yet properly implemented.
What is done here is surprising. I am not a fan of replacing all labels/descriptions in XML by `@text` entries but this is a way to do. What is strange here is that it looks like that `@text` entries look like the default paths that will have been considered even without the `@text`. So it was relatively useless to update all the XML.
**I would suggest to undo these changes from this PR and do it in a separate PR.** This can be done automatically by simply run a maven command. Using this command has the big advantage as it can be rerun later to simply add a new entry.
And regarding the German translations, it is better to update them in Crowdin in a second time. Because if the PR is merged and no manual action is done in Crowdin, there is a risk to loose them in the future.  An exception could be done as this is more or less a first massive injection of translations.

My suggestion: separate PR and simply use the maven command to generate the default translations.

Reply:
- " is not yet properly implemented." - where is a proper way described. 
- "This can be done automatically by simply run a maven command. " - which one?
- "And regarding the German translations, it is better to update them in Crowdin in a second time. " - looking to the volume of translations this is a huge effort. Where is the Crowdin integration described? Where documented that this is a formal requirement?
- "Yes please"
- "My suggestion: separate PR" - again, a huge effort. Everything has to be done manually. I think we should work on another one to make the move to CrowedIn.

Reply:
I supported @markus7017 on the translations, I could also support on moving resources to CrowdIn, but I also recommend the exception for this PR to avoid blocking the process. If the @text statements are not nessesary we could remove them, but maybe @fwolter could also proceed.

Reply:
@lolodomo any advice?

Reply:
@fwolter @lolodomo who do we solve this issue

Reply:
I'd suggest to populate Crowding correctly once this is merged. Also because @ijasan offered his support.